web/packages/teleport/src/services/auth/auth.ts

```
@@ -315,15 +302,53 @@ const auth = {
     return api.post(cfg.api.createPrivilegeTokenPath, {});
   },
 
-  getWebauthnResponse(
+  async getWebauthnResponse(
     scope: MFAChallengeScope,
     allowReuse?: boolean,
     isMFARequiredRequest?: IsMfaRequiredRequest
   ) {
+    // TODO(Joerger): DELETE IN 16.0.0
+    // the create mfa challenge endpoint below supports
+    // MFARequired requests without the extra roundtrip.
+    if (isMFARequiredRequest) {
+      try {
+        const isMFARequired = await checkMfaRequired(isMFARequiredRequest);
+        if (!isMFARequired.required) {
+          return;
+        }
+      } catch {
+        // checking MFA requirement for admin actions is not supported by old
+        // auth servers, we expect an error instead. In this case, assume MFA is
+        // not required. Callers should fallback to retrying with MFA if needed.
```

Can you check for that specific error instead of ignoring all errors?